### PR TITLE
[CONV-16] variables between parantheses must use spaces

### PR DESCRIPTION
**Pull request by Codetaal**

### CODE REVIEW

**Review by AlexAtSea — APPROVED** [review]

**Review by bartvandebiezen — APPROVED** [review]

**Comment by 7img** [comment]
Maybe add another example with a pipe? 
```javascript
{{ productPrice | price }}
```

File: general/README.md [hunk 1]
```diff
@@ -257,3 +257,16 @@ function getList(
 	parameter3
 ) {}
 ```
+
+
+## Spaces
+When working with variables between parantheses, always seperate them by adding spaces.
```
Comment (romypetitjean) [inline general/README.md]: separate 😇

Comment (7img) [reply general/README.md]: And aLways should be always. 👮 
image
^ Weird

File: general/README.md [hunk 2]
```diff
@@ -257,3 +257,16 @@ function getList(
 	parameter3
 ) {}
 ```
+
+
+## Spaces
```
Comment (edgargressie) [inline general/README.md]: See comments above and maybe use the description as title

File: general/README.md (hunk 1 above)
Comment (michaelmaat) [reply general/README.md]: They are called curly braces instead of parantheses ;)

Comment (romypetitjean) [reply general/README.md]: parentheses 😇

**Review by edgargressie — CHANGES_REQUESTED** [review]

**Review by edgargressie — APPROVED** [review]

**Pull request merged**